src/storage/segcache/src/hashtable/mod.rs

```diff
@@ -0,0 +1,685 @@
+// Copyright 2021 Twitter, Inc.
+// Licensed under the Apache License, Version 2.0
+// http://www.apache.org/licenses/LICENSE-2.0
+
+use crate::*;
+use ahash::RandomState;
+
+use rustcommon_time::CoarseInstant as Instant;
+
+const N_ITEM_SLOT: usize = 8;
```

Comment:
Please add a comment saying when chained we lose a slot.

Reply:
This should probably be `N_BUCKET_SLOT` - but let me take a look an add ascii art and blog links to various spots =)

Reply:
Changed the name to be less confusing and added docs for this module